h2o-core/src/main/java/water/H2O.java

```diff
@@ -564,9 +562,6 @@ else if (s.matches("md5skip")) {
       else if (s.matches("quiet")) {
         trgt.quiet = true;
       }
-      else if(s.matches("useUDP")) {
-        trgt.useUDP = true;
```

Comment:
Can we provide an explanatory message here?

Reply:
@michalkurka sure, thanks!